python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -504,6 +505,24 @@ def makeMinimalSchema(filterNameList, *, addCentroid=False,
                 type="Flag",
                 doc="Set if parallax or parallax error is bad",
             )
+            if fullCoordinateCovariance:
+                if not (addProperMotion and addParallax):
+                    warnings.warn("Adding proper motion and parallax covariances without proper motion and "
+                                  "parallax parameters and errors.")
```

Comment:
This seems like a bad idea; why would we ever want someone to be able to do this?

Reply:
I agree that no one will want to do this, but I don't think it's worth forcing a failure if they do, hence the warning.

Reply:
We definitely want this to be a failure, not a warning. No one wanting to do this means it should fail.